core/src/mindustry/editor/EditorTool.java

```diff
@@ -44,6 +51,115 @@ public void touchedLine(MapEditor editor, int x1, int y1, int x2, int y2){
             });
         }
     },
+    rectangle("fill"){
+
+        @Override
+        public void touchedLine(MapEditor editor, int x1, int y1, int x2, int y2){
+            if (x1>x2){
```

Comment:
This formatting is incorrect; all comparisons and assignments should have spaces around the operator. Reformat with the style file mentioned in `CONTRIBUTING.MD`.